1/index.js

```diff
@@ -1 +1,11 @@
-console.log('1!');
+//lesson01--------------------------------------------------
+//chatGPT prompt："javascript DOM作成 li テキスト「これです」"
+//reference:https://www.wakuwakubank.com/posts/306-javascript-dom/#index_id4
+//reference:https://stackoverflow.com/questions/50250214/how-to-use-insertbefore-without-second-parameter
+
+//create list item
+const newList = document.getElementById('newlist'); // get:ID='newlist'
```

Comment:
IMO
If the element is to be used in a JS file, it would be good to add the `js-`prefix to make it explicit that it is to be used in JS.
```suggestion
const newList = document.getElementById('js-new-list'); 
```

Reply:
https://zenn.dev/haniwaman/articles/bf392f397c8db7341881#:~:text=JavaScript%E3%81%A8%E7%B4%90,%E5%A4%89%E6%9B%B4%E3%81%AFOK%EF%BC%89

Reply:
Thank you!
I added a js-prefix!

Thank you for the reference site!

Reply:
https://github.com/megusasuke0/frontendhandson/commit/d1cd14bfa81aaa8d39ce0a8fd4d90cb7be53f627